Makefile

```diff
@@ -1,22 +1,35 @@
 export GO15VENDOREXPERIMENT := 1
 
+
```

Comment:
Maybe add comments for each target will make the usage more clear. What do you think?
And put `.PHONY` on top of each relevant target, to avoid difference between `.PHONY` declaration and targets.